iroh-one/src/config.rs

```diff
@@ -0,0 +1,417 @@
+use anyhow::{bail, Result};
```

Comment:
I can't re-use other configs directly because they all include rpc-client and metrics. The situation is a bit better for p2p since the LibP2PConfig exists and is re-usable here.